certs/readme.md

```diff
@@ -1,39 +0,0 @@
-# Certificates generation
```

Comment:
How does removing this file factor in to the certificates for unit tests?

Reply:
Since we removed the static certs, I think this is not required anymore. This repo purpose is to provide jwt as a plugin for fastify, so I think it's out of scope to include instructions on how to generate certs, but if we want to do that, I think the right place an `example` folder.

Reply:
The static certs is used for test and this document is a guide for ppl to generate their own cert. It should not be removed directly. Placing it in `example` or `/` is fine for me.

Reply:
fair, take a look https://github.com/fastify/fastify-jwt/pull/146/files#diff-3d532493d202967e315d87b13f96266566f53f2036c81493259f5a73838893a4